scripts/kurtosis/network_params.json

```diff
@@ -71,13 +75,13 @@
     ],
     "tx_spammer_params": {
         "tx_spammer_extra_args": [
-            "--txcount=250"
+            "--txcount=1"
```

Comment:
Why/how this helps?

Reply:
The unreasonable spikes were because txcount was 100x too high. txcount=1 generates spike of about 100 transactions every minute as I recall. txcount=250 thus tried to generate 25000 transactions every minute